tests/mongodb/test_mongo_wrapper.py

```diff
@@ -13,7 +13,7 @@ def test_mongo_wrapper_find_as_cursor(mongodb):
 
     count = 0
     for i in cursor:
-        print(i)
+        log.info(i)
```

Comment:
`py.test` will soak up logging output of warning level or less. Is this call important here and is it used by anyone?

https://docs.pytest.org/en/latest/logging.html

It's pretty unconventional to use logging directly in a unit test as the there are other ways of marking behavior.

Consider deleting line, [changing pytest's default logging threshold](https://docs.pytest.org/en/latest/logging.html#caplog-fixture), moving code into the library, or using mocking such that no logging is necessary in testing.

Reply:
Good point here.